samples/cellular/udp/src/main.c

```diff
@@ -239,5 +269,14 @@ int main(void)
 
 	k_work_schedule(&socket_transmission_work, K_NO_WAIT);
 
+#if defined(CONFIG_NRF_MODEM_LIB) && !defined(CONFIG_UDP_DATA_UPLOAD_FOREVER)
```

Comment:
You can remove all the occurrences where we check for `NRF_MODEM_LIB`. This sample runs only on nRF9 series boards, so this is always defined. Keep it a separate commit if you can.